src/package-vsix.js

```diff
@@ -78,24 +79,25 @@ let version = '1.39.1-prel';
         const pck = JSON.parse(originalContent);
         const nlsContent = fs.readFileSync(nlsPath, 'utf-8');
         const nls = JSON.parse(nlsContent);
-        
+
         // note: do change pck.publisher - it's part of the key used to
         // lookup extensions, and so changing it may prevent dependent extensions
-        // to not find it
-        // pck.displayName = nls.displayName || extDisplayName + " (built-in)";
-        pck.displayName = nls.displayName? nls.displayName + " (built-in)" : extDisplayName ;
+        // from finding it
+        pck.displayName = nls.displayName ? nls.displayName + " (built-in)" : extDisplayName;
         pck.description = nls.description || "Built-in extension that adds (potentially basic) support for " + capitalize(pck.name);
         pck.keywords = ["Built-in"];
         pck.repository = repository;
         pck.version = version;
```

Comment:
We should set
```
pck.license = 'SEE LICENSE IN LICENSE-vscode.txt';
```
so the registry knows where to find the license file. An alternative would be to rename the file to the standard name `LICENSE`.

Reply:
done